lib/spec/appliance_console/internal_database_configuration_spec.rb

```diff
@@ -35,10 +35,11 @@
   end
 
   it "#post_activation" do
-    LinuxAdmin.stub(:run).with("chkconfig", :params => {"--add" => "miqtop"})
-    LinuxAdmin::Service.stub(:new => double.as_null_object)
-    ApplianceConsole::ServiceGroup.any_instance.stub(:start_command)
-    LinuxAdmin::Service.should_receive(:new).with(ApplianceConsole::POSTGRESQL_SERVICE).and_return(double(:enable => true))
+    expect_any_instance_of(ApplianceConsole::ServiceGroup).to receive(:enable_miqtop)
+    expect_any_instance_of(ApplianceConsole::ServiceGroup).to(
+      receive(:run_service).with(ApplianceConsole::POSTGRESQL_SERVICE, "enable").and_return(true))
+    expect_any_instance_of(ApplianceConsole::ServiceGroup).to receive(:run_service).at_least(4).times
+    expect_any_instance_of(ApplianceConsole::ServiceGroup).to receive(:run_detached_service).at_least(4).times
```

Comment:
@kbrock I think this test was not very useful even after this change since it's covered by the service group spec..  Unless you think there's some value, I wouldn't be offended if you just :scissors: it.